src/constant/index.js

```diff
@@ -10,7 +10,7 @@ const VERSIONS_GROUPS = {
     STABLE_VERSION: "3.0",
     LEGACY_VERSIONS: ["1.0", "2.0"],
     DEV_PREVIEW_VERSION: "",
-    DEPRECATED_VERSIONS: ["3.1"],
+    DEPRECATED_VERSIONS: [""],
```

Comment:
as mentioned [here](https://text.slack.com/archives/D03919XPNNM/p1682698394034889), I checked and it looks like v3.1 should not be here, so I removed it.